docs/userguide/icu4j/index.md

```diff
@@ -68,41 +67,122 @@ There are a few different ways to download the ICU4J releases.
     *   [ICU Download page](https://icu.unicode.org/download)
     *   Maven repository:
 
-~~~
-<dependency>
-    <groupId>com.ibm.icu</groupId>
-    <artifactId>icu4j</artifactId>
-    <version>68.1</version>
-</dependency>
-
-<dependency>
-    <groupId>com.ibm.icu</groupId>
-    <artifactId>icu4j-charset</artifactId>
-    <version>68.1</version>
-</dependency>
-
-<dependency>
-    <groupId>com.ibm.icu</groupId>
-    <artifactId>icu4j-localespi</artifactId>
-    <version>68.1</version>
-</dependency>
-~~~
+        ```
+        <dependency>
+            <groupId>com.ibm.icu</groupId>
+            <artifactId>icu4j</artifactId>
+            <version>68.1</version>
```

Comment:
Nitpick: newer version?
(74.1, or at least 73.2 ?)

Reply:
Ack. I'm okay with an old version. We shouldn't feel the need to update this during a new version release's BRS tasks. If it's not old now, it will become so after the next version.